conf/local.conf.js

```diff
@@ -7,6 +7,7 @@ exports.config = {
     browserName: 'chrome',
     name: "local_test",
     build: "cucumber-js-browserstack",
```

Comment:
Let's update buildName to `browserstack-build-1` as well
and name to `BStack local cucumber-js` (general sessionName can be like `BStack <local/single/parallel> <fw name>`